commit 6c7944f2c23a5d317250bd9eeba8b1541f9e7bb1 (refs/changes/39/939/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-11-22T14:40:57-08:00 (2 years, 11 months ago)
    
    joyent/node-triton#144 update dashdash dep to get Bash completion fix for sort order of "no completions" completions

diff --git a/CHANGES.md b/CHANGES.md
index e73b81d..eafca8c 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -7,6 +7,16 @@ Known issues:
 
 ## not yet released
 
+- [trentm/node-dashdash#30] Change the output used by Bash completion support to
+  indicate "there are no completions for this argument" to cope with different
+  sorting rules on different Bash/platforms. For example:
+
+        $ triton -p test2 package get <TAB>          # before
+        ##-no -tritonpackage- completions-##
+
+        $ triton -p test2 package get <TAB>          # after
+        ##-no-completion- -results-##
+
 ## 4.14.2
 
 - TOOLS-1592 First workaround for a possible BadDigestError when using
diff --git a/package.json b/package.json
index dd505b7..07d0307 100644
--- a/package.json
+++ b/package.json
@@ -8,7 +8,7 @@
     "backoff": "2.4.1",
     "bigspinner": "3.1.0",
     "bunyan": "1.5.1",
-    "cmdln": "4.1.1",
+    "cmdln": "4.1.2",
     "extsprintf": "1.0.2",
     "lomstream": "1.1.0",
     "mkdirp": "0.5.1",
